packages/mobile/src/navigator/Headers.v2.tsx

```diff
@@ -59,9 +59,10 @@ export const emptyHeader: StackNavigationOptions = {
   headerShown: true,
   headerTitleStyle: [styles.headerTitle, componentStyles.screenHeader],
   headerTitleContainerStyle: {
-    alignItems: 'center',
+    alignItems: 'center' as 'center',
```

Comment:
btw since this is now typed as `StackNavigationOptions` the `as 'center'` should be no longer needed